projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/templates/load/fact/insert/README.md

```diff
@@ -38,6 +38,10 @@ def run(job_input):
     # . . .
 ```
 
+The diagram below is inspired by [this blog](https://www.datacoral.com/blog/data-connectors-101/) and shows how the data from the source view is inserted into the target table, without checking for duplicate data.
```

Comment:
I would rather include the diagram first, because that's the important point you make about the VDK template,
and then add the inspirational article - unless the blogpost is your main point, at which case you need to probably dedicate a paragraph on it to emphasise why.

Reply:
Now that Iva mentioned it . I'd remove the inspired by completely. The inspiration is very tenious at best. The diagram doesn't look at all like the one in the blog post. And also the blog is about ingestion and we are doing transformations. It would just be confusing to users reading it.

Reply:
I believe both of your comments are addressed. My team pointed out that there were some syntax errors that I fixed in the image itself. Let me know if you think we can resolve this and merge the changes to the main branch.

Reply:
Isee It's approved by both of us. Feel free to merge it.